lib/stats_reporter.rb

```diff
@@ -0,0 +1,98 @@
+# generates text for a report on storage root size and object status
+class StatsReporter
+
+  require 'text-table'
+  require 'open3'
+
+  def self.storage_report_text
+    head = %w[filesystem total used pct_used free pct_free]
+    Text::Table.new(head: head, rows: storage_report_lines).to_s
+  end
+
+  def workflow_report_text
+    head = %w[workflow waiting error recent archived]
+    Text::Table.new(head: head, rows: workflow_report_lines).to_s
+  end
+
+  def self.workflow_xml
+    File.open('config/workflows/sdr/preservationIngestWF') { |f| Nokogiri::XML(f) }
```

Comment:
This should be memoized so that, for example, calling `.repository` and `.ingest_wf` doesn't cause the same file to be parsed twice.